src/services/mocks/temperatureServiceMock.js

```diff
@@ -1,4 +1,4 @@
-var DHTState = require('../dhtstate')
+var DHTState = require('../../dtos/dhtstate')
```

Comment:
Maybe this module can help. https://www.npmjs.com/package/rfr

Also I read that if we set $NODE_HOME variable it will consider that one as a root when using require().